src/workspace.coffee

```diff
@@ -47,6 +47,8 @@ class Workspace extends Model
       left: new PanelContainer({location: 'left'})
       right: new PanelContainer({location: 'right'})
       bottom: new PanelContainer({location: 'bottom'})
+      topBar: new PanelContainer({location: 'top-bar'})
+      bottomBar: new PanelContainer({location: 'bottom-bar'})
```

Comment:
Yeah, wasn't too sure about the `bar` name. Any good alternative? 

- `top-full`
- `top-full-width`
- `top-outer`
- `top-edge`
- `top-most`